### ISSUE TITLE
How to disable recreation of map on each reset?

### ISSUE THREAD

**Issue by bkkgbkjb** [MichaelTMatthews/Craftax#35]
Hi all,

thanks for this excellent work

After reading your paper, I wonder if the unexpectedly insignificant performance increase from `PPO-RND` to `PPO` arises from the recreation of 'game'(map and others) on each env reset.

According to your paper, on each env reset, a whole new map and other settings like *Potions and Enchantments* would be changed.

However, in the vanilla `PPO-RND` method, no special measures are taken to cater such a sudden change in dynamics and reward function.

So I wanna ask if we could disable this recreation on 'environment' side and only  reset agent-related  states (health points, items, armours, etc.)  as  what a normal, non-rougelike game does for a game-over.

Do you have any ideas how to achieve this in code? I currently think about cloning env_state from jax then swap it back but am afraid this may damage diversity

Big thanks in advance

**Comment by MichaelTMatthews** [comment]
Hey, I think the easiest way to achieve a fixed map would be to simply fix the seed for world generation. You could do this by adding the line
```
rng = jax.random.PRNGKey(0)
```
to the beginning of `generate_world` to have it always generate the map with seed 0.

**Comment by MichaelTMatthews** [comment]
Closing as seems to be resolved, feel free to reopen if you have further questions
